modules/apps/accessibility/accessibility-test/src/testFunctional/macros/AccessibilityMenu.macro

```diff
@@ -58,7 +58,18 @@ definition {
 			locator1 = "AccessibilityMenu#MODAL_TITLE");
 	}
 
-	@summary = "Default summary"
+	@summary = "It asserts that the user can access the Accessibility Menu modal by the user's profile menu."
+	macro assertCanAccessAccessibilityMenuModalByTheUserProfileMenu() {
+		Navigator.openURL();
+
+		UserBar.gotoDropdownItem(dropdownItem = "Accessibility Menu");
```

Comment:
Oh, I thought I was right...

Because  one is: 
```
	macro assertCanAccessAccessibilityMenuModalByTheUserProfileMenu() {
		Navigator.openURL();

		UserBar.gotoDropdownItem(dropdownItem = "Accessibility Menu");
                         _// It opens the user's profile menu AND accesses the accessibility menu modal_

		AssertElementPresent(
			key_modal_title = "Accessibility Menu",
			locator1 = "AccessibilityMenu#MODAL_TITLE");
	}

```

And another is :

```
	macro assertCannotAccessAccessibilityMenuModalByTheUserProfileMenu() {
		Navigator.openURL();

		UserBar.gotoDropdown();
                       //It accesses the user's profile menu drop-down.

		AssertElementNotPresent(
			key_modal_title = "Accessibility Menu",
			locator1 = "AccessibilityMenu#MODAL_TITLE");
	}
```

If I turn 

```
		Navigator.openURL();

		UserBar.gotoDropdown();
```
Into a macro, I'll have to  add a click on the `assertCanAccessAccessibilityMenuModalByTheUserProfileMenu`.

That's how I see it, but I'll refactor it the way you proposed and we can see what's best later. 🙇🏻‍♂️